File: __tests__/__fixtures__/output/python/python3/multipart-form-data.py
import http.client

conn = http.client.HTTPSConnection("httpbin.org")

payload = "-----011000010111000001101001\r\nContent-Disposition: form-data; name=\"foo\"\r\n\r\nbar\r\n-----011000010111000001101001--\r\n"

headers = { 'Content-Type': "multipart/form-data; boundary=---011000010111000001101001" }

conn.request("POST", "/anything", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: src/targets/node/axios/fixtures/multipart-form-data.js
import axios from 'axios';

const options = {
  method: 'POST',
  url: 'https://httpbin.org/anything',
  headers: {'Content-Type': 'multipart/form-data; boundary=---011000010111000001101001'},
  data: '-----011000010111000001101001\r\nContent-Disposition: form-data; name="foo"\r\n\r\nbar\r\n-----011000010111000001101001--'
};

axios
  .request(options)
  .then(res => console.log(res.data))
  .catch(err => console.error(err));